docs/developer/decision-records/2022-10-10-naming-conventions/README.md

```diff
@@ -0,0 +1,111 @@
+- Accordance to the Eclipse Foundation's guidelines and release processes.
+- Elimination of split packages.
+
+## Approach
+
+Existing standards that are considered in the following are listed below.
+- Apache Maven's [guide to naming conventions on groupId, artifactId, and version](https://maven.apache.org/guides/mini/guide-naming-conventions.html)
+- Oracle's [Java package naming conventions](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html)
+
+### Naming Conventions
+
+#### Gradle Module
+
+All core components of the EDC project are implemented as Gradle multi-module projects. A Gradle module
+is referenced in the `settings.gradle`, e.g.:
+
+```kotlin
+include(":core:data-plane:data-plane-framework")
+```
+
+In this example, the _moduleId_ is `data-plane-framework`. This module implements the "data plane framework"
+that is part of the "data plane" without naming the `core` module. With this, the _moduleId_ is a unique
+concatenation of its location, without preventing the moving and a restructuring of the package.
+
+
+#### Maven Artifact
+
+A Maven artifact is composed of the following components:
+```
+<groupId>.<artifactId>.<version>
+```
+
+The _groupId_ uniquely identifies our project: `org.eclipse.edc.*`. All EDC core components (as of 10.10.2022:
+[connector](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector),
+[identity hub](https://github.com/eclipse-dataspaceconnector/IdentityHub),
+[registration service](https://github.com/eclipse-dataspaceconnector/RegistrationService),
+and [federated catalog](https://github.com/eclipse-dataspaceconnector/FederatedCatalog))
+are published with the same version under this same _groupId_.
+
+The _artifactId_ is the name of the jar without a version. In the EDC project, the _artifactId_ is identical
+to the _moduleId_. For example:
+
+```kotlin
+publishing {
+    publications {
+        create<MavenPublication>("data-plane-framework") {
+            artifactId = "data-plane-framework"
+            from(components["java"])
+        }
+    }
+}
+```
+
+For versioning, see [here](../2022-07-06-release-automation/).
+
+#### Java Package
+
+Following Oracle's [Java package naming conventions](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html),
+package names are written in all lower case. In addition, package names use the reversed domain name,
+for example, `com.example.mypackage` for a package named `mypackage` created by a programmer at `example.com`.
+
+Accordingly, in the EDC project, aligned with the _groupId_, Java packages start with `org.eclipse.edc.*`.
+Corresponding to the _moduleId_, this is followed by a unique path. To remain with our example, the
+class `DataPlaneFrameworkExtension` is located at:
+
+```java
+package org.eclipse.dataspaceconnector.dataplane.framework;
+```
+
+In this context, everything after `org.eclipse.edc` is sorted first by field/discipline and then by scope.
```

Comment:
Instead of "field/discipline", can we say "component"?

Reply:
reformulated